### ISSUE TITLE
Table copy by drag-and-drop to a newly-created SQLiteCypher database causes the database to be saved as non-encrypted 

### ISSUE THREAD

**Issue by MichalLeonBorsuk** [pawelsalawa/sqlitestudio#4120]
### Details
Copying a table from an existing DB to a *new* Crypto DB causes the latter to lose the "crypto status", i.e. it is saved as normal SQLite data.

### Steps to reproduce
1. open a standard SQLite (non-compressed) database as a data source
2. create a new DB of the type "SQLCipher" (here named SQLiteCipherTest.db); give it a password
3. drag a table from standard SQLite to the SQLCipher DB
4. close SQLiteStudio
5. check file type: not encrypted
```
    $ file SQLiteCipherTest.db
    SQLiteCipherTest.db : SQLite 3.x database, last written using SQLite version 3032002
```

### Operating system
Linux xxx 5.4.0-73-generic &82-Ubuntu SMP Wed Apr 14 17:39:42 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### SQLiteStudio version
v3.3.3

**Comment by pawelsalawa** [comment]
Yes, unfortunately currently SQLiteStudio always attaches receiving database (where you drop object) into the source database (from which you drag object), thus source database SQLite library is used. If it was pure SQLite, it will attach the other one also with pure SQLite library.

To fix this, SQLiteStudio needs to start checking first if receiving database uses library other than the default one and in that case switch sides (i.e. attach source db to target db, using encrypting library).

**Comment by pawelsalawa** [comment]
Duplicate of #3767
Fixed with #3767
